dropwizard-testing/src/main/java/io/dropwizard/testing/DropwizardTestSupport.java

```diff
@@ -72,20 +72,30 @@ public void before() {
     }
 
     public void after() {
-        for (ServiceListener<C> listener : listeners) {
+        try {
+            stopIfRequired();
+        }
+        finally {
+            resetConfigOverrides();
+        }
+    }
+
+    private void stopIfRequired() {
+        if( jettyServer != null) {
```

Comment:
Minor code style nitpick: Please remove the leading space inside the parentheses, and add one after `if`.